README.adoc

```diff
@@ -135,7 +180,21 @@ Use the `-u` option to pass in the username `bob` and the password `bobpwd`.
 
 include::{common-includes}/os-tabs.adoc[]
 
-[.tab_content.windows_section.mac_section]
+[.tab_content.windows_section]
+--
+[role=command]
+```
+curl http://localhost:31000/system/properties -u bob:bobpwd
+```
+If the `curl` command is unavailable, then use https://www.getpostman.com/[Postman^].
+Postman enables you to make requests using a graphical interface.
+To make a request with Postman, enter `\http://localhost:31000/system/properties` into the URL bar.
+Next, set the `Authorization` with `Basic Auth` type, and set the `Username` field to `bob` and `Password` field to `bobpwd`. 
```

Comment:
Next, set the `Authorization` with `Basic Auth` type, and set the `Username` field to `bob` and `Password` field to `bobpwd`. 
--->
Next, set the `Authorization` with `Basic Auth` type. Set the `Username` field to `bob` and `Password` field to `bobpwd`.